webapp/components/survey/NodeDefDetails/BasicProps/store/useIsDisplayAsTableDisabled.js

```diff
@@ -0,0 +1,9 @@
+import * as Survey from '@core/survey/survey'
+import * as NodeDef from '@core/survey/nodeDef'
+
+import { useSurvey } from '@webapp/store/survey'
+
```

Comment:
do really need this hook? u just set a property in the useBasiProps?

Reply:
u r right (2)